sdk/src/main/java/io/opentelemetry/sdk/metrics/DoubleCounterSdk.java

```diff
@@ -69,7 +69,7 @@ BoundInstrument newBinding(Batcher batcher) {
     @Override
     public void add(double increment) {
       if (increment < 0) {
-        throw new IllegalArgumentException("monotonic counters can only increase");
+        throw new IllegalArgumentException("Counters can only increase");
```

Comment:
with this throw, we should make sure to unbind above in a finally block.

Reply:
Done.